app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
It wouldn't leave the repos, but it would leave the content (for up to a week).  I think that is a rare occurrence, and its likely that lifecycle environments share a lot of content (so the extra content is probably fairly small).  

Since its only temporary, and the user could run the rake task or cron job manually, i think its okay?  Maybe others disagree?